sdk/storage/azure-storage-common/inc/azure/storage/common/version.hpp

```diff
@@ -14,7 +14,7 @@
 #define AZURE_STORAGE_COMMON_VERSION_MAJOR 12
 #define AZURE_STORAGE_COMMON_VERSION_MINOR 0
 #define AZURE_STORAGE_COMMON_VERSION_PATCH 0
-#define AZURE_STORAGE_COMMON_VERSION_PRERELEASE "beta.6"
+#define AZURE_STORAGE_COMMON_VERSION_PRERELEASE "beta.7"
```

Comment:
Where's the other CL update?

Reply:
something wrong with the pipeline, I guess that's because those pipelines cannot run in parallel.

Reply:
Do these pipelines run in parallel rather than sequentially, one after the other?
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=686888&view=results
![image](https://user-images.githubusercontent.com/6527137/104566550-98e50000-5602-11eb-826f-b77cfdbcbd51.png)

Consider having four independent CI pipelines, one for each of the storage packages, rather than one.

Reply:
There does seem to be an issue with trying to run these CI legs in parallel, all of which trying to update the same branch and PR at the same time, which can cause these types of issues.